CHANGELOG.md

```diff
@@ -43,6 +49,25 @@ corresponding [documentation](/docs/migration/Version_0.1.x_0.3.x.md).
 - Local TXDC Setup Documentation (#618)
 - Feature: Sftp Provisioner and Client (#554)
 
+- Add contract id to data source http call (#732)
+- Support also support releases in ci pipeline
+- Introduce typed object for oauth2 provisioning
+- Add documentation
+- Add test case
+- Add client to omejdn
+- add hydra deployment
+- Configure dynamically HTTP Receiver callback endpoints. (#685)
+- cp-adapter : code review, rollbacke name change (#664)
+- Feature/cp adapter task 355 356 357 (#621)
+- Add Validity Mapping in ContractDefinitionStepDefs class
+- Add feature and create SendAnOfferwithoutConstraints method in class negotiationSteps
+- Add validity attribute in class ContractDefinition
+- Add Validity Mapping in ContractDefinitionStepDefs class
+- Add feature and create SendAnOfferwithoutConstraints method in class negotiationSteps
+- Add validity attribute in class ContractDefinition
+- Local TXDC Setup Documentation (#618)
+- Feature: Sftp Provisioner and Client (#554)
```

Comment:
I think there is something wrong. The points are not doubled. Have a look at the lines (new) 33-50.
Same for the Changed and Fixed sections